app/containers/UnlockEtherContainer.tsx

```diff
@@ -0,0 +1,200 @@
+          over you will be penalized with 10% value of your funds.
+        </p>
+        <div className={styles.section}>
+          <div className={styles.infoBox}>
+            <div className={styles.caption}>Your wallet address:</div>
+            <div className={styles.value}>
+              {this.props.params.address}
+            </div>
+          </div>
+
+          <div className={styles.infoBox}>
+            <div className={styles.caption}>Your NEU balance: </div>
+            <div className={styles.value}>
+              <MoneyComponent value={this.props.neumarkBalance} tokenType={TokenType.NEU} />
+            </div>
+          </div>
+
+          <div className={styles.infoBox}>
+            <div className={styles.caption}>Ether locked: </div>
+            <div className={styles.value}>
+              <MoneyComponent value={this.props.lockedAmountEth} tokenType={TokenType.ETHER} />
+            </div>
+          </div>
+
+          <div className={styles.infoBox}>
+            <div className={styles.caption}>NEU needed to unlock ether funds: </div>
+            <div className={styles.value}>
+              <MoneyComponent
+                value={this.props.neumarkNeededToUnlockEth}
+                tokenType={TokenType.NEU}
+              />
+            </div>
+          </div>
+
+          <div className={styles.infoBox}>
+            <div className={styles.caption}>Free unlock date: </div>
+            <div className={styles.value}>
+              {this.props.unlockDateEth.format("YYYY-MM-DD")}
+            </div>
+          </div>
+        </div>
+        {this.renderSteps()}
+      </div>
+    );
+  }
+
+  private renderSteps() {
+    const neumarkNeededToUnlockAsBI = new BigNumber.BigNumber(this.props.neumarkNeededToUnlockEth);
+    const neumarkBalanceAsBI = new BigNumber.BigNumber(this.props.neumarkBalance);
+
+    const withdrawPossible = neumarkBalanceAsBI.greaterThanOrEqualTo(neumarkNeededToUnlockAsBI);
+    const willBePenalized = moment().isBefore(this.props.unlockDateEth);
+
+    if (!withdrawPossible) {
+      return (
+        <div>
+          <Alert bsStyle="danger">
+            You need{" "}
+            <MoneyComponent
+              value={neumarkNeededToUnlockAsBI.sub(neumarkBalanceAsBI)}
+              tokenType={TokenType.NEU}
+            />{" "}
+            Neumarks more to unlock your account.
+          </Alert>
+        </div>
+      );
+    }
+
+    return (
+      <div className={styles.section}>
+        {willBePenalized
+          ? <Alert bsStyle="danger">
+              Withdrawing funds now will result in penalty of 10% of your funds!
+            </Alert>
+          : <Alert bsStyle="info">Withdrawing funds now will not result in penalty.</Alert>}
+        <h3>Steps to unlock your ether:</h3>
+        <ol>
+          <li>
+            Send transaction to unlock your locked funds:
+            <TxInfo
```

Comment:
Can't we reuse nice box we have in "Other Wallet" on commitment page?
![image](https://user-images.githubusercontent.com/17202864/34074283-246ae928-e2ac-11e7-952f-f93cc8f91761.png)

Reply:
`Neumark smart contract address`:
Zamiast minimum ticket size dac 
![image](https://user-images.githubusercontent.com/17202864/34074297-45d1d072-e2ac-11e7-8e25-1f65d916d805.png)